### PR TITLE
Add Visual Studio 14 Changes

### PR DESCRIPTION
**Pull request by mariuz**

### CODE REVIEW

File: src/gpre/ada.cpp [hunk 1]
```diff
@@ -662,13 +662,13 @@ static void gen_based( const act* action, int column)
...
 		{
-			sprintf(s2, "%s range %"SLONGFORMAT"..%"SLONGFORMAT",\n                            ",
+			sprintf(s2, "%s range %" SLONGFORMAT"..%" SLONGFORMAT",\n                            ",
 					LONG_DCL, dimension->dim_lower, dimension->dim_upper);
 			for (p = s2; *p; p++, q++)
 				*q = *p;
 		}
 
-		sprintf(s2, "%s range %"SLONGFORMAT"..%"SLONGFORMAT") of ",
+		sprintf(s2, "%s range % " SLONGFORMAT"..%" SLONGFORMAT") of ",
```
Comment (asfernandes) [inline src/gpre/ada.cpp]: Space is wrong here after percent and before closing quote.

File: src/jrd/dpm.epp [hunk 2]
```diff
@@ -1100,13 +1100,13 @@ SSHORT DPM_fetch_back(thread_db* tdbb,
...
-		printf("DPM_fetch_back (record_param %"QUADFORMAT"d, lock %d)\n",
+		printf("DPM_fetch_back (record_param %" QUADFORMAT"d, lock %d)\n",
 			rpb->rpb_number.getValue(), lock);
 	}
 	if (debug_flag > DEBUG_READS_INFO)
 	{
-		printf("    record  %"SLONGFORMAT":%d transaction %"SLONGFORMAT
-				  " back %"SLONGFORMAT":%d\n",
+		printf("    record  %" SLONGFORMAT":%d transaction %" SLONGFORMAT
+				  " back % "SLONGFORMAT":%d\n",
```
Comment (asfernandes) [inline src/jrd/dpm.epp]: Space is wrong here after percent and before closing quote.

File: src/jrd/dpm.epp [hunk 3]
```diff
@@ -1168,12 +1168,12 @@ void DPM_fetch_fragment( thread_db* tdbb, record_param* rpb, USHORT lock)
...
 #ifdef VIO_DEBUG
 	if (debug_flag > DEBUG_READS)
-		printf("DPM_fetch_fragment (record_param %"QUADFORMAT"d, lock %d)\n",
+		printf("DPM_fetch_fragment (record_param %" QUADFORMAT"d, lock %d)\n",
 			rpb->rpb_number.getValue(), lock);
 	if (debug_flag > DEBUG_READS_INFO)
 	{
-		printf("    record  %"SLONGFORMAT":%d transaction %"SLONGFORMAT
-				  " back %"SLONGFORMAT":%d\n",
+		printf("    record  %" SLONGFORMAT":%d transaction % "SLONGFORMAT
```
Comment (asfernandes) [inline src/jrd/dpm.epp]: Space is wrong here after percent and before closing quote.

File: src/gpre/ada.cpp (hunk 1 above)
Comment (mariuz) [inline src/gpre/ada.cpp]: fixed

File: src/jrd/dpm.epp (hunk 2 above)
Comment (mariuz) [inline src/jrd/dpm.epp]: fixed

File: src/jrd/dpm.epp (hunk 3 above)
Comment (mariuz) [inline src/jrd/dpm.epp]: fixed

**Comment by asfernandes** [comment]
Merged.

**Pull request closed without merging**